code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2423,11 +2424,14 @@
 	if((prob(10) && method == TOUCH) || method == INGEST)
 		M.contract_disease(new /datum/disease/robotic_transformation(0), 1)
```

Comment:
I had that recommended to me and I couldn't for the life of me get that working without fucking the roburger up.